scripts/deploy_subnet_under_calibration_net/prepare_local.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# A script which resets your local .ipc folder so we can run the deploy.sh script for a local deployment
+#
+# To run:
+# 1. Run `./prepare_local.sh`
+# 2. Run `./deploy.sh`
+#
+# Known issues:
+# - The deploy.sh script does not run "make docker-build" which can cause errors if its outdated on your
+#    machine. To fix this, run `make docker-build` in ipc/fendermint folder and rerun the deploy.sh script
+
+set -eo pipefail
+
+SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+echo "SCRIPT_DIR: $SCRIPT_DIR"
+
+echo "Installing dependencies..."
+cargo install toml-cli
+
+echo "Removing existing .ipc folder"
+rm -rf ~/.ipc
+
+echo "Copying new .ipc folder"
+cp -r $SCRIPT_DIR/.ipc ~/.ipc
+
+echo "Fetching info about newest current deployment contracts"
+hash=$(curl -s https://raw.githubusercontent.com/consensus-shipyard/ipc/cd/contracts/deployments/r314159.json)
+echo $hash | jq
```

Comment:
Why do you call this a `hash`? Is it supposed to be a "HashTable"? I associate it with an actual hash. How about "deployment"?

Reply:
Coming from Ruby?

Reply:
haha, yeah coming from Ruby, I can rename it to something else